d3rlpy/logging/wandb_adapter.py

```diff
@@ -0,0 +1,73 @@
+import wandb
+from typing import Any, Dict, Optional
+from .logger import LoggerAdapter, LoggerAdapterFactory, SaveProtocol
+
```

Comment:
Please add `__all__` in this line.
```py
__all__ = ["LoggerWanDBAdapter", "WanDBAdapterFactory"]
```
And, then, please add an import to this file:
https://github.com/takuseno/d3rlpy/blob/master/d3rlpy/logging/__init__.py